### ISSUE TITLE
Peer Feedback

### ISSUE THREAD

**Issue by ngv557** [NumEconCopenhagen/projects-2020-laerke-jose#5]
Great assignment!
First of all, all the codes worked perfectly on my computer.
The best part of the project was the structure, it was very easy to follow and very intuitive. The tables and figures was very clarifying.
Some improvements could be:
I like the table with Gini-coefficients (gini.sample(10)), but it would be nice to attach a comment to it. Maybe already there reflect on why Gentofte has more inequality than the rest.
Also the codes:
gini['values'][2][:] and map_df.sample(10). It seems unnecessary to include in the project. It doesn’t really clarify anything. 
I would be quite interesting to see the same Gini-time period for Gentofte, as is showed for Langeland.
In conclusion, an interesting assignment.
